ONSdigital/eq-publisher-v3
# Ear 1342 themes

This PR includes updates for the new Themes functionality.

To test:

1. In Author, fill in the themes page and export the JSON.
2. Spin up Publisher V3 and send a POST request at `localhost:9000/publish` with your Author JSON in the body of the request.

The surveyID and legal basis, EQ ID and Form Type for the theme set as 'preview theme' should be used to populate the respective fields in the Publisher output.
